Fix CONTRIBUTING coding style (#485)
OpenZeppelin/cairo-contracts: CONTRIBUTING.md

Before starting development, please [create an issue](https://github.com/OpenZeppelin/cairo-contracts/issues/new/choose) to open the discussion, validate that the PR is wanted, and coordinate overall implementation details.

Also, consider that snake case is used for Cairo development in general due to its strong Python bias.

> This project's opinionated approach consists of using snake_case for all internal functions and camelCasing all exposed functions and parameters.

Reviewer: camelCasing all exposed functions and parameters implies all parameters?, including the ones for internal functions? I think it could be a little confusing.

Author: Yes, great point!

Before starting development, please [create an issue](https://github.com/OpenZeppelin/cairo-contracts/issues/new/choose) to open the discussion, validate that the PR is wanted, and coordinate overall implementation details.

Also, consider that snake case is used for Cairo development in general due to its strong Python bias.

> This project's opinionated approach consists of camelCasing all exposed function names and their parameters:

Reviewer 2: ```suggestion
This project follows our [Extensibility pattern](https://docs.openzeppelin.com/contracts-cairo/extensibility), camelCasing all exposed function names and their parameters:
```

given how contentious the topic is, i'd avoid saying this is opinionated (although it is, but no need to raise it). also the extensibility link early in the section should help shed some light on what "internal" means, otherwise it goes unmentioned
